.github/workflows/workflow.yml

```diff
@@ -104,3 +114,18 @@ jobs:
       if: always()
       with:
         report_paths: 'tests/reports/*.xml'
+
+    - name: build debian package
+      # language=sh
+      run: |
+        sudo apt-get update -q -y
+        sudo apt-get install -q -y git python3 python3-venv python3-pip debhelper devscripts
+        mv dist/*.tar.gz chtools.tar.gz
```

Comment:
python and pip are already present, see `setup-python` version above

Reply:
When I tested this workflow with act, python3* packages weren't present. Maybe the reason is in medium container image instead of a full one, but I suggest being sure on these packages